### PR TITLE
nmap: edit page

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by fazlearefin** [comment]
`nmap` have be authored by a number of individuals over the years. It was lacking coherence within the examples and was also not following tldr style guide. I have not only tried to make the examples coherent but also spent a couple of hours testing the examples throughly using wireshark and multiple hosts to make sure the examples are correct.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/nmap.md [hunk 1]
```diff
@@ -1,37 +1,37 @@
 # nmap
 
 > Network exploration tool and security/port scanner.
-> Some features only activate when Nmap is run with root privileges.
-> More information: <https://nmap.org>.
+> Some features like SYN scan activates only when `nmap` is run with root privileges.
```
Comment (acuteenvy) [inline pages/common/nmap.md:4]:
```suggestion
> Some features (e.g. SYN scan) activate only when `nmap` is run with root privileges.
```

File: pages/common/nmap.md [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
+> More information: <https://nmap.org/book/man.html>.
 
-- Check if an IP address is up, and guess the remote host's operating system:
+- Scan (the top 1000) ports of a remote host with various [v]erbosity levels:
```
Comment (acuteenvy) [inline pages/common/nmap.md:7]:
```suggestion
- Scan the top 1000 ports of a remote host with various [v]erbosity levels:
```

File: pages/common/nmap.md [hunk 3]
```diff
@@ -1,37 +1,37 @@
...
+`sudo nmap -A {{ip_or_hostname1,ip_or_hostname2...}}`
 
-- Scan a specific list of ports (use '-p-' for all ports from 1 to 65535):
+- Scan a specific list of ports (use `-p-` for all ports from 1 to 65535):
 
-`nmap -p {{port1,port2,...,portN}} {{address_or_addresses}}`
+`nmap -p {{port1,port2...}} {{ip_or_host1,ip_or_host2...}}`
 
-- Perform service and version detection of the top 1000 ports using default NSE scripts; writing results ('-oN') to output file:
+- Perform service and version detection of the top 1000 ports using default NSE scripts; writing results (`-oA`) to output files:
```
Comment (acuteenvy) [inline pages/common/nmap.md:23]: Is there any file other than what's specified here (`top-100-ports`)?
```suggestion
- Perform service and version detection of the top 1000 ports using default NSE scripts; writing results (`-oA`) to the output file:
```

File: pages/common/nmap.md [hunk 4]
```diff
@@ -1,37 +1,37 @@
...
+`nmap -sC -sV -oA {{top-1000-ports}} {{ip_or_host1,ip_or_host2...}}`
 
-- Scan target(s) carefully using 'default and safe' NSE scripts:
+- Scan target(s) carefully using `default and safe` NSE scripts:
 
-`nmap --script "default and safe" {{address_or_addresses}}`
+`nmap --script "default and safe" {{ip_or_host1,ip_or_host2...}}`
 
-- Scan web server running on standard ports 80 and 443 using all available 'http-*' NSE scripts:
+- Scan web server running on standard ports 80 and 443 using all available `http-*` NSE scripts:
```
Comment (acuteenvy) [inline pages/common/nmap.md:31]:
```suggestion
- Scan for web servers running on standard ports 80 and 443 using all available `http-*` NSE scripts:
```

File: pages/common/nmap.md [hunk 5]
```diff
@@ -1,37 +1,37 @@
...
 
-`nmap --script "http-*" {{address_or_addresses}} -p 80,443`
+`nmap --script "http-*" {{ip_or_host1,ip_or_host2...}} -p 80,443`
 
-- Perform a stealthy very slow scan ('-T0') trying to avoid detection by IDS/IPS and use decoy ('-D') source IP addresses:
+- Attempt IDS/IPS detection by extremely slow scan (`-T0`), decoy source addresses (`-D`), [f]ragmented packets, random data and other methods:
```
Comment (acuteenvy) [inline pages/common/nmap.md:35]:
```suggestion
- Attempt IDS/IPS detection by using an extremely slow scan (`-T0`), decoy source addresses (`-D`), [f]ragmented packets, random data and other methods:
```

File: pages/common/nmap.md (hunk 3 above)
Comment (fazlearefin) [reply pages/common/nmap.md:23]: @acuteenvy Yes, `-oA` produces 3 output files of different formats (`top-1000-ports.xml`, `top-1000-ports.nmap` and `top-1000-ports.gnmap`) and hence the word "files".

**Review by kbdharun — APPROVED** [review]

**Review by Magrid0 — APPROVED** [review]
Looks good to me

File: pages/common/nmap.md [hunk 6]
```diff
@@ -1,37 +1,37 @@
...
+`sudo nmap -A {{ip_or_hostname1,ip_or_hostname2...}}`
 
-- Scan a specific list of ports (use '-p-' for all ports from 1 to 65535):
+- Scan a specific list of ports (use `-p-` for all ports from 1 to 65535):
 
-`nmap -p {{port1,port2,...,portN}} {{address_or_addresses}}`
+`nmap -p {{port1,port2...}} {{ip_or_host1,ip_or_host2...}}`
 
-- Perform service and version detection of the top 1000 ports using default NSE scripts; writing results ('-oN') to output file:
+- Perform service and version detection of the top 1000 ports using default NSE scripts; writing results (`-oA`) to output files:
```
Comment (acuteenvy) [inline pages/common/nmap.md:23]:
```suggestion
- Perform service and version detection of the top 1000 ports using default NSE scripts, writing results (`-oA`) to output files:
```

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**